service/ziti-tunnel/service/ipc.go

```diff
@@ -498,6 +499,21 @@ func serveIpc(conn net.Conn) {
 
 			//save the state
 			rts.SaveState()
+		case "UpdateTunIpv4":
+			var tunIPv4 string
+			var tunIPv4Mask int
+			var addDns string
```

Comment:
if you want you can just convert this to a boolean right here. that's what happens in IdentityOnOff above

Reply:
I did it this way, because I wanted to make the fields optional. This ipc function will pass the values as string, so I will be able to do an empty check before updating it.